<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="combo_laporan">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hari</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Laporan Per</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>621</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>439</width>
     <height>49</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight SemiConde</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LAPORAN PENJUALAN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Simcard</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_simcard">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Axis</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Telkomsel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Indosat</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Smartfren</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-8</y>
     <width>651</width>
     <height>531</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>picture/bg.jpeg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>combo_laporan</zorder>
  <zorder>label_2</zorder>
  <zorder>tableView</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>combo_simcard</zorder>
  <zorder>pushButton_back</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
